bus-mapping/src/circuit_input_builder/transaction.rs

```diff
@@ -352,9 +367,14 @@ impl Transaction {
         Ok(Self {
             block_num: eth_tx.block_number.unwrap().as_u64(),
             hash: eth_tx.hash,
+            tx_type: TxTypes::get_tx_type(eth_tx),
+            rlp_bytes: eth_tx.rlp().to_vec(),
+            rlp_unsigned_bytes: get_rlp_unsigned(eth_tx),
```

Comment:
`rlp_unsigned_bytes` is the bytes used for signing, and it is also used for calculating l1fee. However, l1 fee cost is base on the pre-EIP155 rlp bytes so the l1 fee obtained from this field is not consistent with l2geth.

Reply:
check the discussion in this issue #515.